client/package.json

```diff
@@ -55,6 +56,7 @@
     "@tmcw/togeojson": "^5.7.0",
     "@trpc/client": "^10.38.0",
     "@trpc/server": "^10.38.0",
+    "@ui-kitten/components": "^5.3.1",
```

Comment:
?

Reply:
tamagui select component hadd a bug i was going to use this instead. removed now!